libs/ml/src/ops/activations/sigmoid.cpp

```diff
@@ -33,7 +33,8 @@ Sigmoid<TensorType>::Sigmoid(SPType const &sp)
 template <typename TensorType>
 std::shared_ptr<OpsSaveableParams> Sigmoid<TensorType>::GetOpSaveableParams()
 {
-  return std::make_shared<SPType>();
+  auto sp = std::make_shared<SPType>();
```

Comment:
what was the point in this change?

Reply:
Point is I forgot to add base class savable params!